src/pk/ecc/ecc_get_curve_by_name.c

```diff
@@ -38,15 +217,27 @@ static int _name_match(const char *left, const char *right)
 int ecc_get_curve_by_name(const char *name, const ltc_ecc_curve **cu)
```

Comment:
then we also need a `ecc_curve_get_name()` which simply returns the first element of the names

Reply:
I am not sure, I do not a use case for `ecc_curve_get_name`